Decent.Minecraft.Client/RailType.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Decent.Minecraft.Client
+{
+    /// <summary>
+    /// The type of rail.
+    /// </summary>
+    public enum RailType : byte
```

Comment:
Should this really be public? Should it even be in Client, or does it belong in Java?